Original patch from: ../4.3.2/270-missing-execinfo_h.patch

-= BEGIN original header =-
Original patch from gentoo: gentoo/src/patchsets/gcc/4.3.1/uclibc/90_all_301-missing-execinfo_h.patch
-= END original header =-

diff -durN gcc-4.3.3.orig/boehm-gc/include/gc.h gcc-4.3.3/boehm-gc/include/gc.h
--- gcc-4.3.3.orig/boehm-gc/include/gc.h	2007-04-23 23:10:09.000000000 +0200
+++ gcc-4.3.3/boehm-gc/include/gc.h	2009-01-27 22:25:37.000000000 +0100
@@ -503,7 +503,7 @@
 #if defined(__linux__) || defined(__GLIBC__)
 # include <features.h>
 # if (__GLIBC__ == 2 && __GLIBC_MINOR__ >= 1 || __GLIBC__ > 2) \
-     && !defined(__ia64__)
+     && !defined(__ia64__) && !defined(__UCLIBC__)
 #   ifndef GC_HAVE_BUILTIN_BACKTRACE
 #     define GC_HAVE_BUILTIN_BACKTRACE
 #   endif
